Makefile

```
@@ -46,25 +44,22 @@ help:
 
 # Dependencies for deployment in an ubuntu/debian linux
 # (lib*-dev merely for building tesserocr with pip)
-# (tesseract-ocr: Ubuntu 18.04 now ships 4.0.0,
-#  but some beta, not the final release,
-#  on which tesserocr 2.4.0 depends;
-#  this downloads a tesseract build from git
-#  and installs it system-wide -
-#  intended for dockerfile and travis,
-#  not recommended for live systems!)
+# (tesseract-ocr: Ubuntu 18.04 now ships 4.0.0
+#  which is unsupported. Add the tesseract-ocr PPA
+#  from Alexander Pozdnyakov which provides 4.1.0.
+#  See https://launchpad.net/~alex-p/+archive/ubuntu/tesseract-ocr
+#  for details.)
 deps-ubuntu:
+	apt-get install -y --no-install-recommends software-properties-common
+	add-apt-repository -y ppa:alex-p/tesseract-ocr
 	apt-get install -y \
 		git \
 		python3 \
 		python3-pip \
 		libtesseract-dev \
 		libleptonica-dev \
 		tesseract-ocr-eng \
-		tesseract-ocr \
-		wget
-	wget -O - https://github.com/nijel/tesseract-ocr-build/releases/download/4.0.0-1/tesseract.tar.xz | tar -xJf -
-	cp -rt /usr .tesseract/*
+		tesseract-ocr
```

by way of https://github.com/OCR-D/ocrd_tesserocr/pull/96#discussion_r353830484, please add 

```suggestion
		tesseract-ocr \
		g++
```

---

Thanks!

---

Done. I try to keep such entries ordered, so I did not add it at the end.